way_to_home/route/models.py

```diff
@@ -1 +1,73 @@
 """This module implements class that represents the route entity."""
+
+from django.db import models, IntegrityError
+from utils.abstract_models import AbstractModel
+from way.models import Way
+from place.models import Place
+
+
+class Route(AbstractModel):
+    """Model for Route entity."""
+
+    way = models.ForeignKey(Way, on_delete=models.CASCADE)
+    start_place = models.ForeignKey(Place, on_delete=models.CASCADE)
+    end_place = models.ForeignKey(Place, on_delete=models.CASCADE)
+    time = models.TimeField()
+    transport_id = models.PositiveIntegerField()
+    position = models.PositiveSmallIntegerField()
+
+    def __str__(self):
+        """Method that returns route instance as string."""
+
+        return f'route from place with id: {self.start_place.id} ' \
+            f'to place with id {self.end_place.id} id'
+
+    def to_dict(self):
+        """Method that returns dict with object's attributes."""
+
+        return {
+            'id': self.id,
+            'time': self.time,
+            'transport_id': self.transport_id,
+            'position': self.position,
+            'way': self.way.id,
+            'start_place': self.start_place,
+            'end_place': self.end_place
+
+        }
+
+    @classmethod
+    def create(cls, way=None, start_place=None, end_place=None,  # pylint: disable=arguments-differ
+               time=None, transport_id=None, position=None):
+        """Method for object creation."""
+
+        route = cls()
+        route.time = time
+        route.transport_id = transport_id
+        route.position = position
+
```

Comment:
User method .save() for save it in database

Reply:
Done